devicetypes/smartthings/qubino-flush-relay.src/qubino-flush-relay.groovy

```diff
@@ -0,0 +1,436 @@
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+	switchEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicSet cmd) {
+	switchEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd) {
+	switchEvents(cmd)
+}
+
+def switchEvents(physicalgraph.zwave.Command cmd) {
+	def value = (cmd.value ? "on" : "off")
+	sendEvent(name: "switch", value: value, descriptionText: "$device.displayName was turned $value")
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd, ep = null) {
+	log.info "SensorMultilevelReport: ${cmd}, endpoint: ${ep}"
+	def result = []
+
+	def map = [:]
+	switch (cmd.sensorType) {
+		case 1:
+			map.name = "temperature"
+			def cmdScale = cmd.scale == 1 ? "F" : "C"
+			map.value = convertTemperatureIfNeeded(cmd.scaledSensorValue, cmdScale, cmd.precision)
+			map.unit = getTemperatureScale()
+			break
+		default:
+			map.descriptionText = cmd.toString()
+	}
+	log.debug "SensorMultilevelReport, ${map}, ${map.name}, ${map.value}, ${map.unit}"
+	handleChildEvent(map)
+	result << createEvent(map)
+}
+
+def on() {
+	log.debug "on"
+	[
+		encap(zwave.basicV1.basicSet(value: 0xFF)),
+		"delay 500",
+		encap(zwave.switchBinaryV1.switchBinaryGet())
+	]
+}
+
+def off() {
+	log.debug "off"
+	[
+		encap(zwave.basicV1.basicSet(value: 0x00)),
+		"delay 500",
+		encap(zwave.switchBinaryV1.switchBinaryGet())
+	]
+}
+
+def ping() {
+	refresh()
+}
+
+def refresh() {
+	refreshChild()
+	def commands = ([
+		encap(zwave.meterV2.meterGet(scale: 0)),
+		"delay 500",
+		encap(zwave.meterV2.meterGet(scale: 2)),
+		"delay 500",
+		encap(zwave.basicV1.basicGet())
+	]) 
+	commands
+}
+
+def createChildDevice(childDthNamespace, childDthName, childDni, childComponentLabel) {
+	try {
+		log.debug "Creating a child device: ${childDthNamespace}, ${childDthName}, ${childDni}, ${childComponentLabel}"
+		def childDevice = addChildDevice(childDthNamespace, childDthName, childDni, device.hub.id,
+				[
+					completedSetup: true,
+					label: childComponentLabel,
+					isComponent: false
+				])
+		log.debug "createChildDevice: ${childDevice}"
+		childDevice
+	} catch(Exception e) {
+		log.debug "Exception: ${e}"
+	}
+}
+
+private refreshChild() {
+	if(childDevices){
+		def childDni = "${device.deviceNetworkId}:2"
+		def childDevice = childDevices.find { it.deviceNetworkId == childDni }
+
+		if (childDevice != null) {
+			sendHubCommand(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 1, scale: 0).format())
+		}
+	}
+}
+
+def handleChildEvent(map) {
+	def childDni = "${device.deviceNetworkId}:" + 2
+	log.debug "handleChildEvent / find child device: ${childDni}"
+	def childDevice = childDevices.find { it.deviceNetworkId == childDni }
+
+	if(!childDevice) {
+		log.debug "handleChildEvent / creating a child device"
+		childDevice = createChildDevice("smartthings", "Qubino Temperature Sensor", childDni, "Qubino Temperature Sensor")
```

Comment:
This isn't the one I wanted you to change.

"Qubino Temperature Sensor" is in the "qubino" namespace.

"Child Temperature Sensor" is in the "smartthings" namespace.

You want to use "qubino" "Qubino Temperature Sensor"